x/bank/module.go

```diff
@@ -227,7 +228,14 @@ type bankInputs struct {
 	Key           *store.KVStoreKey
 }
 
-func provideModule(in bankInputs) (keeper.Keeper, runtime.AppModuleWrapper) {
+type bankOutputs struct {
+	depinject.Out
+
+	BankKeeper keeper.Keeper `key:"cosmos.bank.v1.BankKeeper"`
```

Comment:
what about just using this?
```suggestion
	BankKeeper keeper.Keeper `key:"cosmos.bank.v1.Keeper"`
```

Reply:
I think this approach should be canonical. Just Keeper because bank is already in the package name

Reply:
fixed!